content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,244 @@
+   :align: center
+   :alt: View of a helpdesk team settings page emphasizing the track and bill time settings.
+
+Configure service products
+--------------------------
+
+When the :guilabel:`Time Billing` feature is enabled, a new product is created in the *Sales* app
+called :guilabel:`Service on Timesheets`. This product can be found under :menuselection:`Sales -->
+Products --> Products`. Search for `Service on Timesheets` in the :guilabel:`Search bar`. This is
+the product that will be used when invoicing for services after they have been completed.
+(*post-paid support services*)
+
+Select :guilabel:`Service on Timesheets` from the product page. This reveals the product detail
+form. The product is configured with the :guilabel:`Product Type` set to :guilabel:`Service` and
+the :guilabel:`Invoicing Policy` set to :guilabel:`Based on Timesheets`.
+
+.. image:: track_and_bill/track-bill-product-based-on-timesheets.png
+   :align: center
+   :alt: View of a service product with the invoicing policy set to 'Based on timesheets'.
+
+In order to invoice for support services before the work has been completed (*prepaid support
+services*), a separate product with a different invoicing policy must be created.
+
+To create a new service product, go to :menuselection:`Sales --> Products --> Products` and
+click :guilabel:`New`. This will reveal a blank product detail form.
+
+On the new product form, add a :guilabel:`Product Name`, and set the :guilabel:`Product Type` to
+:guilabel:`Service`. Then, set the :guilabel:`Invoicing Policy` to :guilabel:`Prepaid/Fixed Price`.
+This means an invoice can be generated (and payment can be received) for this product before any
+timesheets entries have been recorded for these services.
+
+.. image:: track_and_bill/track-bill-product-prepaid-fixed.png
+   :align: center
+   :alt: View of a service product with the invoicing policy set to 'prepaid/fixed'.
+
+Finally, set the :guilabel:`Sales Price`, and confirm that the :guilabel:`Unit of Measure` is set
+to :guilabel:`Hours`.
+
+Invoice prepaid support services
+================================
+
+When support services are billed on a fixed price, an invoice can be created before any work is
+completed on the issue. In this case, a service product with the invoicing policy set to
+:guilabel:`Prepaid/Fixed Price` (like the one created above) would be used.
+
+Create a sales order with prepaid product
+-----------------------------------------
+
+To invoice a customer for prepaid support services, first create a sales order (SO) with the
+support services product. To do this, go to :guilabel:`Odoo Dashboard --> Sales --> Orders -->
+Quotations --> New`, which reveals a blank form.
+
+Then, fill out the *quotation* form with the customer information.
+
+Go to the :guilabel:`Order Lines` tab of the quotation and click :guilabel:`Add a Product`. Then,
+select the prepaid services product configured in the steps above. Update the :guilabel:`Quantity`
```

Comment:
I want to italicize _prepaid services product_ here because we are telling the user to select it, but it is not a static element, rather it is a product that the user created in a previous step